dev/fattest.simplicity/src/componenttest/custom/junit/runner/FATRunner.java

```diff
@@ -231,6 +231,10 @@ public void evaluate() throws Throwable {
                     Log.info(c, m, "");
                     Log.info(c, m, "entering " + getTestClass().getName() + "." + method.getName());
                     Log.info(c, m, "");
+                    if (RepeatTestFilter.isAnyRepeatActionActive()) {
+                        Log.info(c, m, "current repeat action: " + RepeatTestFilter.getRepeatActionsAsString());
+                        Log.info(c, m, "");
+                    }
```

Comment:
```suggestion
                    if (RepeatTestFilter.isAnyRepeatActionActive()) {
                        Log.info(c, m, "current repeat action: " + RepeatTestFilter.getRepeatActionsAsString());
                        Log.info(c, m, "");
                    } else {
                        Log.info(c, m, "");
                    }
```
suggestion: reduce number of blank lines in log